manifests/0.1/aws-ebs-csi-operator.v0.1.clusterserviceversion.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     repository: https://github.com/openshift/aws-ebs-csi-driver-operator
     createdAt: 2020-03-24:13:08:00
 
-    apiVersion: csi.ebs.aws.com/v1alpha1
+    apiVersion: csi.ebs.openshift.io/v1alpha1
```

Comment:
`ebs.aws.csi.openshift.io`? 

- `csi.openshift.io` sould be the common preffix. Maybe we could add `storage` to let people who don't understand "csi" that it's storage.
- I'd prefer the cloud / backend name in the driver name, `pd` or `disk` doesn't tell much,  `pd.gce` and `disk.azure` does.

Reply:
+1 for `ebs.aws.csi.storage.openshift.io`.

Reply:
at some point having a very large string as groupName is going to bite us.  IIRC - we had to shorten the groupName in LSO because when combined with CRD name, it was too large to fit in some of OLM related validations (don't remember precisely which one.. :( )

the groupName is mostly detail that user shouldn't have to worry about and having both csi and storage in it is kinda redundant. Also I think having both ebs and aws is redundant. I prefer - `ebs.csi.openshift.io` .. :(